spdy/session_test.go

```diff
@@ -504,18 +504,20 @@ func SpawnClientServerTest(t *testing.T, host string, client ClientRoutine, serv
 	endServer := make(chan bool)
 	listenWait := make(chan bool)
 
+	listener, listenErr := net.Listen("tcp", host)
```

Comment:
Passing in the host as an argument shouldn't be needed any more, just fill this in with "localhost:0"